AndroidTest/app/build.gradle

```diff
@@ -55,10 +40,13 @@ android {
     sourceSets {
         main {
             java.srcDirs = ['src/main/java',
-                            "$scriptLocation/../../sync-core/src/test/java/",
-                            "$scriptLocation/../../sync-core/src/main/java/",
-                            "$scriptLocation/../../sync-android/src/test/java/",
-                            "$scriptLocation/../../sync-android/src/main/java/"]
+                            "$scriptLocation/../../sync-core/src/main/java",
```

Comment:
Do we need to build this code into the app if it's also included as the test code?